Cargo.toml

```
@@ -32,6 +33,11 @@ git = "https://github.com/holochain/holochain"
 rev = "ea026def01d1573d5e0edfb7f4e3e9453f88c43e"
 package = "holochain_types"
 
+[dependencies.holochain_types]
```

Would be best to have the dependency name correspond.

```suggestion
[dependencies.holochain_zome_types]
```

---

This is actually required to build correctly.